addons/spreadsheet/static/tests/global_filters/global_filter_helper.test.js

```diff
@@ -502,13 +502,24 @@ test("getFacetInfo for date values", async () => {
         id: "1",
     };
     const env = {};
-    for (const [period, label] of Object.entries(RELATIVE_PERIODS)) {
+    const relativeFilterTypes = [
```

Comment:
nitpick, we'll probably never update this variable

`globalFilterDateRegistry.getAll().filter(x => !x.isFixedPeriod)` ?
it would also test `this_month`; `this_year` etc which I think fall under the same category?